### DISCUSSION TITLE
Unable to start Detox test - problem at environment.js

### DISCUSSION THREAD

**Discussion by dejanovicnenad** [wix/Detox#2982] [Q&A]
I just started learning Detox, but I got stuck with some error for what Im not sure how to resolve.

After executing command `detox test -c ios` I get error showing me the part of the code from `environment.js` file. I never actually touched that file myself. 

![detox_problem_1](https://user-images.githubusercontent.com/8507831/133460081-ed421894-9ef7-4d35-b645-6e6242169e70.png)

Let me know if some more info is needed to debug! Hope someone will help.
Thanks!

**Comment by lukebrandonfarrell** [comment]
What is your version of `jest`, `jest-circus`, `jest-junit`, `jest-react-native` and `detox` ?

**Reply by dejanovicnenad** [reply]
Ah sorry, I resolved this problem myself but forgot to add a comment for a solution...

The solution in my case was adding this at the end of `package.json`:

```
   "babel": {
       "presets": [
         [
           "@babel/preset-env",
           {
             "targets": {
               "esmodules": false
             }
           }
         ]
        ]
    } 
```

and running `yarn add @babel/preset-env --dev` at the end...
